Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/Annotation/AddIshAnnotation.cs

```diff
@@ -1,5 +1,5 @@
 ﻿/*
-* Copyright (c) 2014 All Rights Reserved by the SDL Group.
+* Copyright © 2014 All Rights Reserved by the RWS Group for and on behalf of its affiliates and subsidiaries.
```

Comment:
We said not to change the copyright in the codebase...and now you changed this I am wondering what the date of the copyright should be since it is 2014 everywhere?